mafiasi/dashboard/templates/dashboard/index.html

```diff
@@ -13,10 +13,12 @@
 {% is_snowing as is_snowing_val %}
 <div class="row {% if is_first_of_april_val %}first-of-april{% endif %}">
     <div id="dashboard" class="col-md-8">
-        <h2>{% trans "Services" %}</h2>
-        {% if is_snowing_val %}
-            <img id="santa-hat-services-deco" src="{% static "img/santa-hat.png" %}" alt="santa-hat" />
-        {% endif %}
+        <h2>
+            {% trans "Services" %}
+            {% if is_snowing_val %}
+                <img id="santa-hat-services-deco" src="{% static "img/santa-hat.png" %}" alt="santa-hat" />
+            {% endif %}
+        </h2>
```

Comment:
By placing the santa-hat inside the title, we can position it in relation to it.